exercise_0/terminalMode/TerminalMode.cpp

```diff
@@ -0,0 +1,16 @@
+//
+// Created by emyrmak on 7/3/2019.
+//
+
+#include <iostream>
+#include "TerminalMode.h"
+
+
+void TerminalMode::run() {
```

Comment:
FileMode::run returns an interger value.
Why did you introduce different behaviour in TerminalMode?